go-controller/pkg/clustermanager/egressip_controller.go

```diff
@@ -1047,6 +1047,64 @@ func (eIPC *egressIPClusterController) reconcileEgressIP(old, new *egressipv1.Eg
 	return nil
 }
 
+// syncEgressIPs This method takes care syncing stale data in the egress ip status
+// with cloud private ip config upon master reboot cases. cloud private ip config
+// entry would have been deleted when master was down whereas egress ip status
+// was not updated for the deleted entry in an error scenario. Hence this method
+// ensures egress ip status is upto date with available cloud private ip config
+// entry.
+func (eIPC *egressIPClusterController) syncEgressIPs(ips []interface{}) error {
+	if !util.PlatformTypeIsEgressIPCloudProvider() {
+		return nil
+	}
+	cloudPrivateIPConfigMap, err := eIPC.getCloudPrivateIPConfigMap()
+	if err != nil {
+		return fmt.Errorf("syncEgressIPs unable to get cloud private ip config: %w", err)
+	}
+	egressIPs, err := eIPC.kube.GetEgressIPs()
```

Comment:
Why are we getting the egressips here if it is what is being received in this mehtod call?